extensions/custom-footer/src/Extension.jsx

```diff
@@ -13,22 +13,38 @@ export default function Extension() {
 
   return (
     <InlineLayout columns={['auto', 'fill']}>
-      <InlineStack spacing="extraTight" blockAlignment="center">
-        <Link to={storefrontUrl}>Home</Link>
-        <Icon source="chevronRight" size="extraSmall" />
-        <Link to={new URL('/collections', storefrontUrl).href}>Shop</Link>
-        <Icon source="chevronRight" size="extraSmall" />
-        <Text appearance="subdued">Checkout</Text>
+      <InlineStack spacing="extraTight" blockAlignment="center" accessibilityRole="orderedList">
+        <InlineStack accessibilityRole="listItem" blockAlignment="center" spacing="extraTight">
+          <Link to={storefrontUrl}>Home</Link>
+          <Icon source="chevronRight" size="extraSmall" />
```

Comment:
not sure what the correct way is for indicating this is a visual separator - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role doesn't seem to be appropriate here since it's not really separating sections of content

Reply:
I don't believe its needed here. Our separator are presentational only. The items are already somewhat separated within the list.